public/catalog/v0.1.0/openapi.json

```diff
@@ -1050,16 +1163,6 @@
             "readOnly": true,
             "title": "Support URL",
             "example": "The URL for finding support for the portfolio item"
-          },
-          "workflow_ref": {
-            "type": "string",
```

Comment:
@mkanoor I see that `workflow_ref`, `owner` columns are removed but I don't see where they're added back in.

Reply:
@syncrou Added the missing properties